Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the runtime image
-FROM mcr.microsoft.com/dotnet/aspnet:9.0.5-bookworm-slim AS runtime
+FROM mcr.microsoft.com/dotnet/aspnet:9.0.8-bookworm-slim AS runtime
```

Comment:
FYI, the latest version can be found here:
https://mcr.microsoft.com/en-us/artifact/mar/dotnet/aspnet/tags